src/qnngds/design.py

```diff
@@ -335,7 +335,9 @@ def __init__(
         self.chip_w = chip_w
         self.chip_margin = chip_margin
         self.N_dies = N_dies
-        self.die_w = die_w
+        self.die_size = unit_die_size
```

Comment:
do you still need these if they get passed into the die parameters?

Reply:
This class is for any user creating a new design, technically the die_parameters class is to simplify the code for contributors so yes a design should still start by fixing all of these paarmeetrs (without having to create first a die_parameetrs class and then pass it to the design class)